QUEST_LV_0100.tsv

```diff
@@ -684,7 +684,7 @@ QUEST_LV_0100_20150317_000683	You got it. Thank you so much.{nl}You saved lots o
 QUEST_LV_0100_20150317_000684	{memo X}We can suppress the vicious energy if the magic of the tomb gets unleashed.{nl}I need the activating stone that is located inside the Guardians.
 QUEST_LV_0100_20150317_000685	The manual for the secret place
 QUEST_LV_0100_20150317_000686	$We hid the secret treasures for the Revelator who would come someday.{nl}Find and absorb the Guardians' hidden energy to open the secret door.
-QUEST_LV_0100_20150317_000687	The hiding place of the brother I used to serve is located at the abondoned farm up there.{nl}Please get the diary of the brother there.
+QUEST_LV_0100_20150317_000687	$The hideout of a brother I used to serve is located at the Abandoned Farm up north.{nl}Please get the diary of my brother there.
```

Comment:
The hideout that I used to serve with my brother is located at the Abandoned Farm up north. ?